Update README.me include kubectl commands (#2)
thepolicylab-projectportals/projectportal-email-service: README.md

# projectportal-email-service

Project Portal email service will run on the production internal cluster, `bkpi-ranch`.

A test version has also been set up on the qa internal cluster, `qbkpi-ranch`.

To access either cluster use the following command 

> ```kubectx bkpi-ranch``` or ```kubectx qbkpi-ranch```

Reviewer: when I run `kubectx qbkpi-ranch` then `kubens` I get: `error: the server doesn't have a resource type "namespaces" error getting namespace list`

> To check which namespace you are in, use ```kubens```, make sure you are in the `tpl-projectportal` namespace in either cluster.

Reviewer: ```suggestion
Make sure you are in the `tpl-projectportal` namespace (in either cluster). Use `kubens` to see your current namespace and `kubens tpl-projectportal` to switch to the `tpl-projectportal` namespace.
```

> The Project Portal email service is a cronjob that is scheduled to run every ... , to check the cronjobs, use ```kubectl get cronjobs```

Reviewer: I get `No resources found in tpl-projectportal namespace.` back when I run this. Is that expected right now? (If so it's fine!)
